crowdsec-docs/docs/appsec/installation.md

```diff
@@ -0,0 +1,130 @@
+
+<!-- @tko fix collection name -->
+
+```
+cscli collections install crowdsecurity-waap/cisa-virtual-patching
+```
+
+Installing this collection brings the relevant appsec-rules (attack signatures), and the `crowdsecurity/vpatch` appsec-configuration (configuration for the appsec component).
+
+# Configuration: Application Security Component
+
+The appsec component is available as a data-source in crowdsec, and allows us to expose a port that the remediation engine / bouncer will send requests to, and gets verdict from. To enable it, we need to add the data source. This can be done by editing `/etc/crowdsec/acquis.yaml` or adding a new data source file in `/etc/crowdsec/acquis.d/`. We are going to add an aplication security component focused on detecting and blocking the exploitation of well-known vulnerabilities (virtual patching):
+
+```bash
+mkdir  -p /etc/crowdsec/acquis.d
+cat > /etc/crowdsec/acquis.d/appsec.yaml << EOF
+listen_addr: 127.0.0.1:4242
+appsec_config: crowdsecurity/vpatch
+source: appsec
+labels:
+  type: appsec
+EOF
+```
+
+The important lines are `listen_addr` that indicates on which interface/port the service listens too, and `appsec_config` that tells which configuration the appsec component will run.
+
+Restart crowdsec:
+
+```bash
+systemctl restart crowdsec
+```
+
+You should be able to see crowdsec starting the aplication security component in the logs (`/var/log/crowdsec.log`):
+
+```
+INFO[2023-12-05 09:16:31] 1 appsec runner to start                      type=appsec
+INFO[2023-12-05 09:16:31] Starting Appsec server on 127.0.0.1:4242/     type=appsec
+INFO[2023-12-05 09:16:31] Appsec Runner ready to process event          type=appsec uuid=3b80fefe-6665-4f81-8567-a2a7f09a706a
+```
+
+and actively listening on the port:
+
+```bash
+# netstat -laputen | grep 4242    
+tcp        0      0 127.0.0.1:4242          0.0.0.0:*               LISTEN      0          6923691    779516/crowdsec     
+
+```
+
+# Configuration : Remediation component
+
+<!-- @kka fix version -->
+At the time of writting, only the crowdsec nginx bouncer supports native integration with the AppSec component, and requires version >= X.Y.Z.
+
+[If you don't have the bouncer installed, do it now !](https://docs.crowdsec.net/u/bouncers/nginx)
+
+To enable the integration, you simply have to add a `APPSEC_URL` parameter to the existing nginx bouncer remediation configuration :
+
+> /etc/crowdsec/bouncers/crowdsec-nginx-bouncer.conf
+```
+...
+APPSEC_URL=http://127.0.0.1:4242
+...
+```
+
+:warning: The remediation component authenticates to the AppSec component using the same API KEY it uses to authenticate to LAPI. Your bouncer must thus have a valid api key for the LAPI the security engine running the AppSec component is connected to :warning:
```

Comment:
[too long and lets skip "thus"]

The remediation component uses the same API key for both AppSec and LAPI communication.
Make sure your bouncer have a valid API [optionally: "and is properly connected to the LAPI."]